types/html-to-text/index.d.ts

```diff
@@ -357,10 +313,27 @@ export interface FormatOptions {
      * (Only for: `table`, `dataTable` formatter.) Number of empty lines between data table rows.
      */
     rowSpacing?: number | undefined;
+    /**
+     * A string to be inserted in place of a tag.
+     */
+    string?: string | undefined;
+    /**
+     * String prefix to be inserted before inline tag contents.
+     */
+     prefix?: string | undefined;
+    /**
+     * String suffix to be inserted after inline tag contents.
+     */
+     suffix?: string | undefined;
```

Comment:
These might need "Only for" comment for consistency with other doc strings.

They are only for specific formatters: https://github.com/html-to-text/node-html-to-text/blob/5aa6b573b940c35173f2d4c38ce652ecf98851a8/packages/html-to-text/README.md?plain=1#L239-L241

Reply:
Updated.